app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -48,10 +48,10 @@ def self.selected_remote(idv_session:)
       if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
          IdentityConfig.store.in_person_proofing_enabled &&
          idv_session.service_provider&.in_person_proofing_enabled
-        idv_session.skip_doc_auth == false
+        idv_session.skip_doc_auth_from_how_to_verify == false
```

Comment:
I’m concerned that the changes to this method will cause problems during deployment.  When I check the [50/50 state docs](https://handbook.login.gov/articles/manage-50-50-state.html#rename-a-field),  it says in the first deploy to read from both fields with `new_name || old_name` and then in the second deploy to remove references to the old name.  

In one deploy, this change reads from only the new variable name and removes references to the old variable name.

Reply:
I did some local manual testing of the hybrid handoff page and I think I found an error.

1. I started the app running locally on this branch
2. I navigated to the hybrid handoff page
3. While leaving the hybrid handoff page open in my browser, I killed the app and restarted it running on main
4. The hybrid handoff page reloaded (without me clicking anything) and displayed the “how to verify page” which comes before the hybrid handoff page.

I made a screenrecording and uploaded it to our team google drive [here.](https://drive.google.com/file/d/1-Qp_orL0jvTlKjjmJ3D7rRYEzxiEop3e/view?usp=drive_link)

Reply:
Thanks for this, we were able to somewhat (not consistently) reproduce this so I went ahead and added back the original check. I also created follow up ticket to remove this secondary check once this is deployed.

Reply:
@KeithNava I again followed the below steps 

1. I started the app running locally on this branch
2. I navigated to the hybrid handoff page
3. While leaving the hybrid handoff page open in my browser, I killed the app and restarted it running on main
4. The hybrid handoff page reloaded (without me clicking anything) and displayed the “how to verify page” which comes before the hybrid handoff page.

Again, I made a screenrecording and uploaded it to our team google drive as [LG-13006Sep2550:50StateBug.mov](https://drive.google.com/file/d/1DG6cJy4QlnwJKoekD9hlRO6brpbweJNm/view?usp=drive_link).

Please let me know if you'd like to pair on this.  If you want, we could try these steps on your local setup and see if you encounter the same bug.

Reply:
Looks like this is happening because the hybrid_handoff controller (https://github.com/18F/identity-idp/blob/main/app/controllers/idv/hybrid_handoff_controller.rb#L51) is expecting `skip_doc_auth` to be set which it wouldn't be when coming from this branch since it's being removed. 

I'm not exactly sure how to get around this given our strategy around removing all instances of skip_doc_auth in this PR - Unless we preemptively change the logic to expect either `skip_doc_auth` or `skip_doc_auth_from_how_to_verify` in this logic. 🤔

Reply:
Since you added `idv_session.skip_doc_auth_from_how_to_verify == false || idv_session.skip_doc_auth == false` to line 51 of the hybrid_handoff_controller, I wouldn't expect there to be a problem here.  The code is following the instruction in [50/50 state documentation](https://handbook.login.gov/articles/manage-50-50-state.html#rename-a-field) about reading from both the old and new field. 🤔

Reply:
Hi, as I noted in my [comment](https://github.com/18F/identity-idp/pull/11043#discussion_r1736926851) at the top of this thread:

> When I check the [50/50 state docs](https://handbook.login.gov/articles/manage-50-50-state.html#rename-a-field), it says in the first deploy to read from both fields with `new_name || old_name` and then in the second deploy to remove references to the old name.
In one deploy, this change reads from only the new variable name and removes references to the old variable name.

This pattern persisted in the how_to_verify_controller.  I modified the how_to_verify_controller to read from both fields with `new_name || old_name`.  After that change, I no longer see an error when I follow the below steps.

1. I started the app running locally on this branch
2. I navigated to the hybrid handoff page
3. While leaving the hybrid handoff page open in my browser, I killed the app and restarted it running on main.  I was able to navigate forwarding in the IPP flow.  Success!

I took a screenrecording.  Let me know if it would be helpful to see.

Below is a git diff of the change.  
```
diff --git a/app/controllers/idv/how_to_verify_controller.rb b/app/controllers/idv/how_to_verify_controller.rb
index 233d6165d..0cd089dda 100644
--- a/app/controllers/idv/how_to_verify_controller.rb
+++ b/app/controllers/idv/how_to_verify_controller.rb
@@ -34,12 +34,12 @@ module Idv
       if result.success?
         if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
           idv_session.opted_in_to_in_person_proofing = false
-          idv_session.skip_doc_auth_from_how_to_verify = false
+          idv_session.skip_doc_auth_from_how_to_verify = false || idv_session.skip_doc_auth = false
           redirect_to idv_hybrid_handoff_url
         else
           idv_session.opted_in_to_in_person_proofing = true
           idv_session.flow_path = 'standard'
-          idv_session.skip_doc_auth_from_how_to_verify = true
+          idv_session.skip_doc_auth_from_how_to_verify = true || idv_session.skip_doc_auth = true
           redirect_to idv_document_capture_url
         end
       else
@@ -63,7 +63,7 @@ module Idv
           idv_session.service_provider&.in_person_proofing_enabled
         end,
         undo_step: ->(idv_session:, user:) {
-                     idv_session.skip_doc_auth_from_how_to_verify = nil
+                     idv_session.skip_doc_auth_from_how_to_verify = nil || idv_session.skip_doc_auth = nil
                      idv_session.opted_in_to_in_person_proofing = nil
                    },
       )
````